dp/cloud/python/magma/radio_controller/tests/unit/test_active_mode_controller_service.py

```diff
@@ -378,20 +414,29 @@ def test_should_send_data_if_cbsd_needs_deletion(self):
             with_state(self.registered). \
             with_desired_state(self.registered). \
             with_registration('some'). \
+            with_max_ibw(1000). \
+            with_carrier_aggregation(True). \
+            with_available_frequencies([3500, 3750]). \
             deleted(). \
             build()
         self.session.add(cbsd)
         self.session.commit()
 
-        config = ActiveModeCbsdBuilder(). \
+        am_cbsd = ActiveModeCbsdBuilder(). \
             with_id(SOME_ID). \
             with_state(Registered). \
             with_desired_state(Registered). \
             with_registration('some'). \
             deleted(). \
             with_category('b'). \
-            build()
-        expected = State(cbsds=[config])
+            with_grant_settings(
+            GrantSettings(
+                grant_redundancy_enabled=True,
+                carrier_aggregation_enabled=True,
+                max_ibw_mhz=1000,
+                available_frequencies=[3500, 3750],
```

Comment:
if its Hz, then maybe lets make those values reflect the actual Band 48, so 3.5 GHz is 3500000 in Hz

Reply:
Actually those are masks, so 3500 is in fact `110110101100`, which means:
3560, 3565, 3575, 3585, 3590, 3600, 3605